embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/storage/PayloadStorageServiceImpl.kt

```diff
@@ -148,9 +148,11 @@ class PayloadStorageServiceImpl(
         if (removalCount < 0) {
             return false
         }
-        val removals = input
-            .sortedWith(storedTelemetryComparator)
-            .takeLast(removalCount)
+        val removals = input.sortedWith(
```

Comment:
Oh interesting, so there's no "descending with comparator" method? That kind of sucks... it means we are maintaining a different prune sort order than delivery :/